lib/PostMessageTransport.js

```diff
@@ -57,8 +79,20 @@ define(function (require, exports, module) {
      */
     function send(idOrArray, msgStr){
         var win = _iframeRef.contentWindow;
+        var msg = JSON.parse(msgStr);
+
+        if(msg && msg.params && msg.params.expression) {
+            msg.params.expression = resolveLinks(msg.params.expression);
+        }
+
+        msgStr = JSON.stringify(msg);
 
         win.postMessage(msgStr,"*");
+
+        if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
```

Comment:
Do we really want to call reload if we just navigate to a non-HTML file?  Should we only do this if we're being asked to go to an HTML page?